cmd/crank/render/runtime_docker.go

```diff
@@ -278,6 +300,39 @@ func (r *RuntimeDocker) createContainer(ctx context.Context, cli *client.Client)
 	return rsp.ID, containerAddr, errors.Wrap(err, "cannot start Docker container")
 }
 
+func (r *RuntimeDocker) getPullOptions() (typesimage.PullOptions, error) {
+	// Resolve auth token by looking into config file
+	named, err := reference.ParseNormalizedNamed(r.Image)
+	if err != nil {
+		r.log.Debug("Image is not a valid reference", "image", r.Image)
```

Comment:
the pattern we have now of returning an empty `PullOptions{}` and an error, then the caller will log any errors is good for me, so let's follow it consistently.  i.e., here there is no need to log at DEBUG anymore. The caller will log it for us :)

Reply:
Missed it. Fix it now.